RFS/src/main/java/com/rfs/version_os_2_11/IndexCreator_OS_2_11.java

```diff
@@ -28,7 +29,8 @@ public Optional<ObjectNode> create(ObjectNode root, String indexName, String ind
         // Assemble the request body
         ObjectNode body = mapper.createObjectNode();
         body.set("aliases", indexMetadata.getAliases());
-        body.set("mappings", indexMetadata.getMappings());
+        // Working around for missing OS_1_3 definition
```

Comment:
Thanks for the comment, but I think I'm even more confused with it.  Why do we care about OS 1.3 in a class specific to OS 2.11?  This feels important because it seems like its pointing to a bigger concept in the code under review I just don't understand.

Reply:
Since we aren't using a factory to produce the creators, in the `FullTest.java` we aren't actually using a OS 1.3 set of creators.  The reason the test failed is that in 1.3 the mapping definition is not an object, but an array.  I'm happy to disable the tests if you'd prefer.

Reply:
Per conversation - the issue here is that the snapshot format is different for 1.3 than 2.11, and we're trying to use the 2.11 constructs in our tests to support 1.3 without actually writing the code required to formally support 1.3 (e.g. `IndexMetadataData_OS_1_3`, `IndexCreator_OS_1_3`, etc).  A simple fix is to just revert the change in the 2.11 index creator so it works for 2.11 snapshots and disable 1.3 test support in the tests given we don't currently support 1.3 in reality.  We can then circle back to support 1.3 properly later.  Alternatively we could add the formal support now but that seems like overkill.  Happy with either approach, but breaking our abstraction to informally support tests in this way doesn't seem like the the right balance for me.

Reply:
I'll disable the test case, using `RFS Dynamically select transformer based on target cluster type` for follow up [1].

-[1] https://opensearch.atlassian.net/browse/MIGRATIONS-1827